{"name": "test_invalidLogin", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'dashboard' in 'https://orangehrm-demo-6x.orangehrmlive.com/securityAuthentication/retryLogin'\n +  where 'https://orangehrm-demo-6x.orangehrmlive.com/securityAuthentication/retryLogin' = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"0dc7d12b77e994df9ddfa3bdf5e9e622\")>.current_url", "trace": "test_setup = None\n\n    @allure.description(\"Validate OrangeHRM with invalid credentials\")\n    @allure.severity(severity_level=\"NORMAL\")\n    def test_invalidLogin(test_setup):\n        driver.get(\"https://orangehrm-demo-6x.orangehrmlive.com/\");\n        driver.find_element_by_id(\"txtUsername\").clear();\n        driver.find_element_by_id(\"txtPassword\").clear();\n        enter_username(\"Admin\");\n        enter_password(\"admin1234\")\n        log(\"Clicking login button\")\n        driver.find_element_by_id(\"btnLogin\").click();\n        try:\n>           assert \"dashboard\" in driver.current_url\nE           assert 'dashboard' in 'https://orangehrm-demo-6x.orangehrmlive.com/securityAuthentication/retryLogin'\nE            +  where 'https://orangehrm-demo-6x.orangehrmlive.com/securityAuthentication/retryLogin' = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"0dc7d12b77e994df9ddfa3bdf5e9e622\")>.current_url\n\n..\\ValidateStringOps\\operations\\test_login.py:40: AssertionError"}, "description": "Validate OrangeHRM with invalid credentials", "steps": [{"name": "Entering Username as 'Admin'", "status": "passed", "parameters": [{"name": "username", "value": "'Admin'"}], "start": 1584707037283, "stop": 1584707037479}, {"name": "Entering Password as 'admin1234'", "status": "passed", "parameters": [{"name": "password", "value": "'admin1234'"}], "start": 1584707037479, "stop": 1584707037788}, {"name": "'Clicking login button'", "status": "passed", "parameters": [{"name": "message", "value": "'Clicking login button'"}], "start": 1584707037788, "stop": 1584707037789}], "attachments": [{"name": "Invalid Credentials", "source": "d60812e6-736b-46a7-b75d-97501d0c786c-attachment.png", "type": "image/png"}], "start": 1584707028836, "stop": 1584707045168, "uuid": "e36f65a3-615a-4bbc-b435-ca22a8a86d2c", "historyId": "70685c0582137de6951d235415e388a2", "testCaseId": "5ce9b12a715c01bd75d928071be98103", "fullName": "operations.test_login#test_invalidLogin", "labels": [{"name": "severity", "value": "NORMAL"}, {"name": "parentSuite", "value": "operations"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "hp-PC"}, {"name": "thread", "value": "5896-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "operations.test_login"}]}